docs/API.md

```diff
@@ -156,6 +156,41 @@ This parameter is a string that are an IPv4 or IPv6. It's validated with the fol
  - IPv6 : `/^([0-9A-Fa-f]{1,4}:[0-9A-Fa-f:]{1,}(:[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})?)|([0-9A-Fa-f]{1,4}::[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})$/`
 
 
+### Language tag
+
+Basic data type: string
+
+A string of A-Z, a-z and underscores matching the regular expression
+`/^[a-z]{2}(_[A-Z]{2})?$/`.
+
+The set of valid *language tags* is further constrained by the
+[LANGUAGE.locale] property.
+* If the *language tag* is a five character string, it needs to match a *locale
+  tag* in LANGUAGE.locale.
+* If the *language tag* is a two-character string, it needs to match the
+  first two characters of exactly one *locale tag* in LANGUAGE.locale.
+  (So that it is unambiguous which *locale tag* is matched.)
+
+E.g. if LANGUAGE.locale is "en_US en_UK sv_SE", all the valid *language tags*
```

Comment:
Could you add a link on LANGUAGE.locale ?

Reply:
Not linking here follows Wikipedia's policy on repeated links (since LANGUAGE.locale is already linked a few lines above):
https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Linking#Duplicate_and_repeat_links

I like that guideline. It feels quite measured to me. Transferring their guideline to our context, I'd say "Language tag" passes for a glossary entry.

Thoughts on this?

Reply:
The idea is interesting. However I am wondering if this applies to documentation, as you would like to make it easier for readers to jump around.